src/core/Form/HintText/HintText.md

```diff
@@ -1,32 +1,44 @@
-HintText is a component that generates an accessible hint text for an adjacent element. Give the ID of the HintText as an `aria-describedby` value to the related component. Applicable form components in the library already have HintText as an integral part.
+The `<HintText>` component creates an accessible instruction text for an adjacent element.
+
+Under the hood, the `suomifi-ui-components` library uses this component extensively in form components.
+
+Examples:
+
+<ul>
+  <li><a href="/#/Components/HintText?id=basic-use">Basic use</a></li>
+</ul>
+
+<div style="margin-bottom: 40px">
+  <a href="/#/Components/HintText?id=props--methods">Props & methods</a>
+</div>
+
+### Basic use
+
+Provide the id of the HintText as an `aria-describedby` value to the related component.
 
 ```js
-import React from 'react';
 import { HintText } from 'suomifi-ui-components';
 
-<HintText>This is a hint regarding an adjacent input</HintText>;
+<div>
+  <label htmlFor="custom-input">Name</label>
+  <HintText id="hint-text" style={{ marginBottom: '10px' }}>
+    Please enter your first and last name
```

Comment:
This feels a bit wrong because it wants first and last name into one field. 
Could it be just "Please enter your first name" ?

Reply:
This is now fixed in the latest commit